src/database/migrations/0014-alter-base_fields-description.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE base_fields
+  ADD COLUMN description VARCHAR NOT NULL DEFAULT '';
+
+ALTER TABLE base_fields ALTER COLUMN description DROP DEFAULT;
```

Comment:
Yeah! (I can't claim credit of course but it seemed like a pretty elegant solution; thanks internet!)

This might be a rare place where a comment would make sense just to highlight exactly what's going on for future readers.